elements/lisk-codec/benchmark/buffer_concat.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+// Buffer.concat x 1,263,976 ops/sec ±6.44% (74 runs sampled)
+
+const { Suite } = require('benchmark');
+const { randomBytes } = require('crypto');
+const suite = new Suite();
+const buf1 = randomBytes(1024);
+const buf2 = randomBytes(1024);
+
+suite
+  .add('Buffer.concat', () => {
+    Buffer.concat([buf1, buf2]);
```

Comment:
I think this is not needed? if we gonna keep this, probably nice to test with more elements and also with totalLength and without totalLength for the second parameter on the `Buffer.concat`

Reply:
ok should i remove this now?

Reply:
Either way it's fine